ci/ci/build.py

```diff
@@ -885,18 +887,18 @@ def build(self, batch, code, scope):  # pylint: disable=unused-argument
 echo create user secret...
 cat > sql-config.json <<EOF
 {{
-  "host": "10.80.0.3",
-  "port": 3306,
+  "host": "$HOST",
+  "port": $PORT,
   "user": "{self.user_username}",
   "password": "$USER_PASSWORD",
-  "instance": "db-gh0um",
-  "connection_name": "hail-vdc:us-central1:db-gh0um",
+  "instance": "$INSTANCE",
+  "connection_name": "hail-vdc:us-central1:$INSTANCE",
   "db": "{self._name}"
 }}
 EOF
 cat > sql-config.cnf <<EOF
 [client]
-host=10.80.0.3
+host=$HOST
```

Comment:
You do need to add the database server config to the create_job at the end of this build function.  Github won't let me make a comment there.